templates/Security/login.html.twig

```diff
@@ -2,82 +2,66 @@
-                            <div class="description">Enter the password that accompanies your username/Email.</div>
-                        </div>
 
-                        <div>
-                            <label class="login-page-text" style="display:inline; float:none; vertical-align:15%;">
-                                <a href="{{ path('pelagos_app_ui_account_password') }}">Forgot Your Password or Password Expired?</a>
-                            </label>
-                        </div>
-                        <div>
-                            <label class="login-page-text" style="display:inline; float:none; vertical-align:15%;">
-                                <a href="{{ path('pelagos_app_ui_account_forgotusername') }}">Forgot Your Username?</a>
-                            </label>
-                        </div>
-                    </div>
-                    {# <input type="hidden" name="_csrf_token" value="{{ csrf_token('authenticate') }}"> #}
-                    <div class="row btn-row">
-                    <input class="btn-submit form-submit" id="edit-submit" name="op" value="LOGIN" accesskey="l" type="submit">
-                    <input class="btn-reset" name="reset" accesskey="c" value="CLEAR" type="reset">
-                    </div>
-                </form>
-                </div>
-                <div id="sidebar">
-                    <p class="fl-panel fl-note fl-bevel-white fl-font-size-80">You have reached the GRIIDC site.</p>
-                    <p>Accounts are governed by rules of Texas A&M University–Corpus Christi. Accounts are issued to individuals and shall not be shared.</p>
-                    <p>User authentication is required to access these pages.</p>
-                    <p>If you need to access GRIIDC pages and are not a registered user, please <a href="{{ path('pelagos_app_ui_account_default') }}">request an account</a>.</p>
-                    <p>Texas A&amp;M University–Corpus Christi</p>
-                    <ul>
-                        <li>Unauthorized use is prohibited.</li>
-                        <li>Usage may be subject to security testing and monitoring.</li>
-                        <li>Abuse is subject to criminal prosecution.</li>
-                        <li>No expectation of privacy except as otherwise provided by applicable privacy laws.</li>
-                    </ul>
-                    <p>These warnings are required in accordance with Texas Administrative Code 202 as administered by the Texas Department of Information Resources.  Reference Chapter 33, Texas Penal Code.</p>
-                    <p></p>
-                </div>
-            </div>
-        </div> <!-- main table -->
+{% if error %}
+    <div role="alert" class="w-75 mx-auto" style="width: 66%;">
+        <div class="border border-t-0 border-red-400 rounded-b bg-red-100 px-4 py-3 text-red-700">
+            Sorry, login not accepted due to {{ error.message }}.
```

Comment:
The only reason/error message given is "Invalid Credentials" so you may reword this a bit.
It only gives that message not to give any hints. It'll never say: Bad password, or unknown user.